all.jsonnet

```diff
@@ -110,10 +110,28 @@ local qf = t.queryFrontend(commonConfig {
     q.service.metadata.namespace,
     q.service.spec.ports[1].port,
   ],
-  splitInterval: '12h',
-  maxRetries: 10,
-  logQueriesLongerThan: '10s',
+  splitInterval: '24h',
+  maxRetries: 5,
+  logQueriesLongerThan: '5s',
```

Comment:
These were changed on purposed to different values than the defaults, to show that overwriting the values actually works, which is kind of the entire point of the `all.jsonnet` in total :wink: :blush: 
Happy to add this as a comment to the top of the file. Wdyt?

Reply:
Oh, got it. Thanks for the clarification! 
I was wondering why this changed yesterday :rofl: I can update these values back.

Reply:
Yes, that would be great and maybe even change values in `all.jsonnet` to different ones that the defaults, haha. :) Doesn't need to be too crazy though.